pkg/cmd/server/origin/ensure.go

```diff
@@ -71,7 +71,9 @@ func (c *MasterConfig) ensureOpenShiftInfraNamespace() {
 			Out:              ioutil.Discard,
 			RoleClient:       c.PrivilegedLoopbackOpenShiftClient.ClusterRoles(),
 		}
-		if err := reconcileRole.RunReconcileClusterRoles(nil, nil); err != nil {
+		if err := kclient.RetryOnConflict(kclient.DefaultBackoff, func() error {
```

Comment:
I think you tried to return the native `err` from the command to support this case, but you still have to inspect the aggregate to handle multiple conflicts resulting in multiple conflict errs in an aggregate.